src/odbc/unittests/connect.c

```diff
@@ -11,6 +11,9 @@ init_connect(void)
 }
 
 #ifdef _WIN32
+#ifndef SQL_NOUNICODEMAP
```

Comment:
I'll take another look, thanks.

Reply:
Ah, right... at least as of 1.0, Unicode builds of this test otherwise try to call `SQLGetPrivateProfileStringW` with unsuitable arguments, yielding several warnings -- and a link error when building the ODBC library and tests in driverless mode.

Reply:
Good point, `CHKDriverConnect` could otherwise misfire.  It somehow worked for me without even any warnings, but I'll still look into reworking this change (while still accounting for the possibility that `SQLGetPrivateProfileString` should redirect to `tds_SQLGetProfileString`).